esbuilder/lit-connect-modal/esbuild.js

```diff
@@ -0,0 +1,26 @@
+const { build } = require('esbuild');
+const path = require('path');
+const fs = require('fs');
+
+const entryPoint = path.resolve(__dirname, 'src/modal.js');
+const outputDir = 'packages/auth-browser/src/lib/connect-modal/modal.ts';
+
+build({
+  entryPoints: [entryPoint],
+  bundle: true,
+  // minify: true,
+  sourcemap: false,
+  outfile: outputDir,
+  globalName: 'LitConnectModal',
+  loader: {
+    '.svg': 'dataurl',
+    '.css': 'text',
+  },
+  sourceRoot: './',
+  format: 'esm',
```

Comment:
this might be a problem for cjs and umd projects? what is the scope of this bundling operation?

Reply:
it's being re-bundle by nx, tested on both umd and cjs works